### DISCUSSION TITLE
How to add lyrics on Jellyfin 10.9.1?

### DISCUSSION THREAD

**Discussion by 1P1E10** [jeffvli/feishin#615] [General]
What's the instructions to add lyrics for a song?

**Comment by iiPythonx** [comment]
You can use tools like [lrcget](https://github.com/tranxuanthang/lrcget), [picard](https://picard.musicbrainz.org/), or [beets](https://beets.readthedocs.io/en/latest/plugins/lyrics.html) just to name a couple. lrcget would more then likely be the easiest option to setup and get going.

If you wanted to do it manually, you could grab lyrics from a place like [LRCLib](https://lrclib.net) and embed them yourself using Jellyfin's web metadata editor.
